scripts/binary-install.js

```diff
@@ -7,8 +7,9 @@ const process = require('node:process')
 const { existsSync, mkdirSync } = require('fs')
 const { join, resolve } = require('path')
 const { spawnSync } = require('child_process')
+const { Readable } = require('node:stream') // Required for stream conversion
+const { fetch } = require('undici') // Keep undici for fetch
```

Comment:
`fetch` should be defined without using `undici`, so this line and the new `undici` can be removed.